code/messaging/src/main/java/com/adobe/marketing/mobile/aepuitemplates/SmallImageTemplate.kt

```diff
@@ -28,7 +29,7 @@ import com.adobe.marketing.mobile.aepuitemplates.utils.AepUITemplateType
  * @property dismissBtn The details for the small image template dismiss button.
  */
 data class SmallImageTemplate(
-    val id: String,
+    val propositionId: String,
```

Comment:
We should let this remain an `id` so as to make it generic and not closely coupled with Messaging.